essential_reading/DataSources/Data_S3.md

```diff
@@ -0,0 +1,223 @@
+all of the translation to interact with S3.
+
+```python
+m = fs.get_mapper('s3://noaa-nwm-retrospective-2-1-zarr-pds/chrtout.zarr')
+# 'm' looks like a file as far as Python is concerned.
+# It is now a stand-in for those functions that require a file as argument.
+g = zarr.convenience.open_consolidated(m) # read zarr metadata for named file.
+print(g.tree())
+```
+
+Output:
+
+```text
+/
+ ├── crs () |S1
+ ├── elevation (2776738,) float32
+ ├── feature_id (2776738,) int32
+ ├── gage_id (2776738,) |S15
+ ├── latitude (2776738,) float32
+ ├── longitude (2776738,) float32
+ ├── order (2776738,) int32
+ ├── streamflow (367439, 2776738) int32
+ ├── time (367439,) int64
+ └── velocity (367439, 2776738) int32
+```
+
+:::{warning}
+
+We are deliberately demonstrating using commands that read **only** the zarr metadata from the
+object storage -- **not the full data set**.
+
+The above example is a very, _very_, **very** large dataset, which you don't want to load over the network to your desktop.
+Execute full data read operations only if this notebook is being hosted and run out of the same AWS center where the data lives.
+
+:::
+
+:::{sidebar}
+
+The good news about some of the larger science-oriented libraries (xarray, dask, pandas, zarr, etc), is that
+they can automatically handle the `fsspec` operations for you **IF YOUR ACCESS IS ANONYMOUS**.
+This is a convenience, but is a special case for read-only data where `anon=True` and `requester_pays=False`.
+
+Note that this is a feature of
+[specific libraries](https://filesystem-spec.readthedocs.io/en/latest/#who-uses-fsspec),
+and doesn't work everywhere.
+
+Because it isn't universally available, and only applies to anonymous reads, examples in HyTEST will
+always explicitly plumb `fsspec` 'longhand' using `get_mapper()`. You may see example code elsewhere
+that takes the shortcut if it is available.
+
+:::
+
+
+## Credentialed Access
+
+For some data storage within the HyTEST workflows, access will not be anonymous.
+Permissions are set by the owners of that data, using credentials assigned to an AWS 'profile'.
+
+Profile credentials are usually stored outside of the Python program (typically in a master file in
+your `HOME` folder on the compute/jupyter server).
+You need to have this set up beforehand, and is usually achieved by copying specific credential files into the right spot.
+A common mechanism to handle the profile configuration is with the `aws`
+[command line interface](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/configure/index.html).
+
+To create a new AWS profile:
+
+```sh
+> aws configure --profile osn-renci
+AWS Access Key ID :
+AWS Secret Access Key :
+Default region name : us-east-1
+Default output format: json
+```
+
+The first two prompts will ask for key/security information you were assigned for access to object storage.
+Answer last questions with "us-east-1" (region) and "json" (format).
```

Comment:
```suggestion
The default region for the OSN pod should be "us-east-1" because this is the region in which the pod is physically located, and default output format should be "json".
```